src/test/java/com/example/lures/LureServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package com.example.lures;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class LureServiceTest {
+    @InjectMocks
+    LureService lureService;
+    @Mock
+    LureMapper lureMapper;
+
+    @Test
+    public void 存在するルアーを全件取得すること() {
+        List<Lure> lureList = List.of(
+                new Lure(1, "Balaam300", "Madness", 300, 168),
+                new Lure(2, "JointedCRAW178", "GANCRAFT", 178, 56));
+        doReturn(lureList).when(lureMapper).findAll();
+        List<Lure> actual = lureService.findLure((String) null);
+        assertThat(actual).isEqualTo(lureList);
+        verify(lureMapper, times(1)).findAll();
+    }
+
+    @Test
+    public void URLにクエリ文字列を指定した場合に指定したクエリ文字列を含むルアーが返されること() {

```

Comment:
ServiceのテストではHTTPリクエストは意識せず業務ロジックに着目した言葉を使うほうがよいですね。
```suggestion
    public void 引数に指定した文字列を含むルアーの情報が返されること() {

```

Reply:
コメントありがとうございます。
修正させていただきます。